otsdaq/ConfigurationInterface/Database_configInterface.cc

```diff
@@ -34,7 +34,7 @@ constexpr auto default_entity     = "OTSROOT";
 //==============================================================================
 DatabaseConfigurationInterface::DatabaseConfigurationInterface()
 {
-#ifdef DEBUG_ENABLE
+// #ifdef DEBUG_ENABLE
```

Comment:
I think I'd prefer the #ifdef over the if(0), if it's conflicting make it ARTDAQ_DATABASE_DEBUG_ENABLE

Reply:
back to ifdef